org-linker.el

```diff
@@ -68,10 +74,14 @@ the list."
 (defun org-linker-search-interface (callback)
   "Setup the helm-org-ql search interface.
 Call CALLBACK with a marker to target heading."
-  (add-to-list 'helm-org-ql-actions (cons "super-link-temp" callback) nil)
-  (helm-org-ql (org-linker-get-search-buffers))
-  (pop helm-org-ql-actions))
-
+  (let* ((boolean 'and)
+         (helm-input-idle-delay helm-org-ql-input-idle-delay)
+         (source (helm-org-ql-source (org-linker-get-search-buffers)
+                                     :name "org-linker target")))
+    ;; You'll need a better action name here
+    (setcdr (assoc 'action source) callback)
+    (helm :prompt (format "Query (boolean %s): " (-> boolean symbol-name upcase))
```

Comment:
There is currently no dependency on dash.el. Maybe you could rewrite this without `->` ?

Reply:
You are right. I've fixed that.